src/clj/ataru/information_request/information_request_service.clj

```diff
@@ -64,6 +66,7 @@
     (start-email-job job-runner connection information-request)
     (audit-log/log {:new       information-request
                     :operation audit-log/operation-new
+                    :session   session
                     :id        virkailija-oid})
```

Comment:
Tässä `:id` pitäis ehkä olla pikemminkin `(:application-key information-request)`.